lib/TaskProcessing/SummaryProvider.php

```diff
@@ -122,11 +122,17 @@ public function process(?string $userId, array $input, callable $reportProgress)
 			try {
 				$completions = [];
 				if ($this->openAiAPIService->isUsingOpenAi() || $this->openAiSettingsService->getChatEndpointEnabled()) {
+					$summarySystemPrompt = 'Summarize the following text in a single paragraph and in the same language as the text.';
```

Comment:
Sorry i have to nitpick again, depending on the length of the input a single paragraph may be too short to include the relevant information :/

Reply:
If we put a cap on the length via the chunk size, then this shouldn't be too much of a problem; also, the advice from OpenAI says that the prompt should be as specific as possible. But even after some testing without the `in a single paragraph and`, the output length didn't change too much for me either, so maybe it's okay to leave it out for now.